source/manage/cloud-vpc-private-connectivity.rst

```diff
@@ -0,0 +1,33 @@
+Cloud VPC Private Connectivity
```

Comment:
it will be nice to have a diagram which gives a high level network topology of how things are connected.

Reply:
The problem with this is that might vary a lot based on the case I have around 10 possible designs in the doc